File: threedollar-domain/src/main/java/com/depromeet/threedollar/domain/user/domain/faq/FaqCreator.java
package com.depromeet.threedollar.domain.user.domain.faq;

import com.depromeet.threedollar.common.docs.ObjectMother;
import lombok.AccessLevel;
import lombok.NoArgsConstructor;

@ObjectMother
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public class FaqCreator {

    public static Faq create(String question, String answer, FaqCategory category) {
        return Faq.newInstance(category, question, answer);
    }

}

File: threedollar-domain/src/testFixtures/java/com/depromeet/threedollar/domain/user/domain/faq/FaqCreator.kt
package com.depromeet.threedollar.domain.user.domain.faq

import com.depromeet.threedollar.domain.user.domain.TestFixture

@TestFixture
object FaqCreator {

    @JvmStatic
    fun create(
        question: String,
        answer: String,
        category: FaqCategory
    ): Faq {
        return Faq.builder()
            .question(question)
            .answer(answer)
            .category(category)
            .build()
    }

}